tutorials/vr/vr_starter_tutorial/vr_starter_tutorial_part_one.rst

```diff
@@ -275,7 +275,7 @@ the same script, so it doesn't matter which you use first. With ``VR_Controller.
     var teleport_raycast
 
     # A constant to define the dead zone for both the trackpad and the joystick.
-    # See (http://www.third-helix.com/2013/04/12/doing-thumbstick-dead-zones-right.html)
+    # See <https://web.archive.org/web/20191208161810/http://www.third-helix.com/2013/04/12/doing-thumbstick-dead-zones-right.html>
```

Comment:
Neither the previous nor the changed markup make much sense here, it's a GDScript comment.